lib/airbrake/rack/notice_builder.rb

```diff
@@ -62,7 +62,7 @@ def add_context(notice)
           context[:action] = @controller.action_name
         end
 
-        notice[:context].merge!(@user.to_hash) if @user
+        notice[:context].merge!(@user.data) if @user
```

Comment:
Curious, why not `to_hash`? I think I prefer that to `as_json` because I always thought `as_json` is a slight misnomer, but it's a convention, and it's fine with me :smile: